custom_conf.py

```diff
@@ -212,6 +212,11 @@
 # that you want to link to:
 # manpages_url = "https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html"
 
+############################################################
+### -- Options for PDF output ------------------------------
+############################################################
```

Comment:
Are there "typical" options users might want to customise here?
Then we should add them (commented out), or at least a comment on what users can do here.

Otherwise we should remove the section, it seems confusing to have an empty section in here.

Reply:
Yeah, there's a few that I know some teams are using but they're all different. I'm sure they can sort it out themselves though. The common one requires some additional overhead so I'll just remove this as suggested.